addons/survey/models/survey_user_input.py

```diff
@@ -457,6 +462,18 @@ def _choice_question_answer_result(self, user_input_lines, question_correct_sugg
         else:
             return 'skipped'
 
+    def _simple_choice_question_answer_result(self, user_input_lines, question_correct_suggested_answers, question_incorrect_scored_answers):
+        correct_user_input_answer = user_input_lines.filtered(lambda line: line.answer_is_correct and not line.skipped).mapped('suggested_answer_id')
+        incorrect_user_input_answer = user_input_lines.filtered(lambda line: not line.answer_is_correct and not line.skipped).mapped('suggested_answer_id')
+        if correct_user_input_answer in question_correct_suggested_answers:
+            return 'correct'
+        elif incorrect_user_input_answer in question_incorrect_scored_answers:
+            return 'partial'
+        elif incorrect_user_input_answer:
+            return 'incorrect'
+        else:
+            return 'skipped'
```

Comment:
I mean we only have 1 answer for those right? So this can be simplified to:

```py
# only 1 answer here
user_input_line = user_input_lines[0]
user_answer = user_input_line.suggested_answer_id if not user_input_line.skipped else self.env['survey.question.answer']
if user_answer in question_correct_suggested_answers:
    return 'correct'
elif user_answer in question_incorrect_scored_answers:
    return 'partial'
elif user_answer:
    return 'incorrect'
else:
    return 'skipped'
```